cartpost-service/src/main/java/com/example/cartpostservice/cart/controller/CartController.java

```diff
@@ -0,0 +1,34 @@
+package com.example.cartpostservice.cart.controller;
+
+
+import com.example.cartpostservice.cart.controller.dto.response.CartItemsGetResponse;
+import com.example.cartpostservice.common.dto.EmptyResponse;
+import com.example.cartpostservice.common.dto.ResponseDto;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/carts")
+public class CartController implements CartApi {
+
+
+    @Override
+    @GetMapping("/items")
+    public ResponseEntity<ResponseDto<CartItemsGetResponse>> getCartItems(@RequestHeader(name = "X-CODE") String xCode) {
+        return null;
+    }
+
+    @Override
+    @DeleteMapping("/{item_code}")
```

Comment:
컨벤션 정할 때 [케밥 케이스](https://developer.mozilla.org/en-US/docs/Glossary/Kebab_case)를 따르기로 했었습니다!
```suggestion
    @DeleteMapping("/{item-code}")
```

Reply:
오타가... 많네요..ㅠㅠ
수정하겠습니다